app/server.rb

```diff
@@ -0,0 +1,41 @@
+require 'hobbit'
+require 'hobbit/contrib'
+require_relative 'custom-hobbit'
+require 'data_mapper'
+
+class Server < Hobbit::Base
+  include Hobbit::Mote
+  include Hobbit::Session
+
+
+
+  # # # # # # # # # # # # # # # #
```

Comment:
nice try but this would have been so much nicer and more explicit just to save it in a descriptive file name and required it.